montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -59,8 +82,11 @@ def load(self, solver, **kwargs):
         solver.set_ref_wave(montblanc.constants.C/tf.getcol('REF_FREQUENCY')[0])
 
         # Get the baseline antenna pairs and correct the axes
-        ant1 = tm.getcol('ANTENNA1').reshape(ntime,nbl)
-        ant2 = tm.getcol('ANTENNA2').reshape(ntime,nbl)
+        ant1 = tm.getcol('ANTENNA1').reshape(file_ant_shape)
+        ant2 = tm.getcol('ANTENNA2').reshape(file_ant_shape)
+	if (data_order == ORDER_BL_TIME):
+		ant1.transpose(ant_transpose)
+		ant2.transpose(ant_transpose)
```

Comment:
Come to think of it, it's just misleading naming. They should have named the method ``transposed()`` and not ``transpose()``.